epub33/rs/index.html

```diff
@@ -384,14 +384,23 @@ <h3>Network access</h3>
 			<section id="sec-epub-rs-external-links">
 				<h3>External links</h3>
 
-				<p data-tests="#pub-external-links">Reading systems SHOULD open links that resolve outside the <a>EPUB
-						publication</a> in a new browser instance to ensure that the browser's security and privacy
-					controls are available to users.</p>
+				<p>When a link has an <code>http</code> or <code>https</code>
+					<a data-cite="url#concept-url-scheme">scheme</a> [[url]], reading systems:</p>
 
-				<p>Although links to external web sites and resources are commonly found in <a>EPUB content
-						documents</a>, these are not the only sources. For example, if a reading system provides access
-					to <a data-cite="epub-33#sec-link-elem">linked records</a> [[epub-33]] in the <a>package
-						document</a> metadata, it should similarly open the links in a new browser instance.</p>
+				<ul>
+					<li id="sec-external-links-consent">SHOULD obtain the user's consent before opening the link.</li>
+					<li data-tests="#pub-external-links" id="sec-external-links-open">SHOULD open the link in a new browser instance to ensure that the browser's security and privacy
+						controls are available to users.</li>
+				</ul>
+
+				<p>For links that have schemes that require a helper application to load, reading systems SHOULD obtain
+					the user's consent and follow platform guidance for opening the helper application.</p>
```

Comment:
While creating the relevant epub test modification, I have added a test for the user consent requirement.

```suggestion
				<p id="ec-external-links-consent" data-tests="#pub-external-links_consent">For links that have schemes that require a helper application to load, reading systems SHOULD obtain
					the user's consent and follow platform guidance for opening the helper application.</p>
```